src/openforms/js/components/admin/form_design/form-creation-form.js

```diff
@@ -116,6 +116,7 @@ const initialFormState = {
   submitting: false,
   logicRules: [],
   priceRules: [],
+  pricingLogic: 'static',
```

Comment:
This should be nested inside the form (since it is an attribute on the form model). Otherwise it won't automatically be sent to the backend!